tests/backup_helper.go

```diff
@@ -6933,3 +6933,74 @@ func ValidateCustomResourceRestores(ctx context1.Context, orgID string, resource
 	}
 	return nil
 }
+
+// func CheckBackupStatus return backup status and reason for a particular backup object.
+func CheckBackupStatus(backupName string, ctx context1.Context) (api.BackupInfo_StatusInfo_Status, string, error) {
```

Comment:
I feel this should be part of backupDriver

Reply:
done.